dependency-suppression/cve-suppressed.xml

```diff
@@ -38,6 +38,7 @@
    ]]></notes>
         <filePath regex="true">.*\/pulsar-client-all-2.11.2.jar\/.*\/org.apache.avro\/.*\/pom\.xml</filePath>
         <cve>CVE-2021-43045</cve>
+        <cve>CVE-2023-37475</cve>
```

Comment:
For Reference: https://github.com/din-global/schedule/actions/runs/5723787793/job/15509007910?pr=17